client/faucet/TODO

```diff
@@ -0,0 +1,4 @@
+* better error message when request params is invalid
```

Comment:
Might be better to have these as Github issues rather than in a TODO file in the codebase

Reply:
good idea